m4_sessions/1803-inning/group_6.md

```diff
@@ -1,13 +1,21 @@
 Group 6
 
-Group Members:
+Group Members: Jorge PerezGa, Parker Lindley, Jeff Bender
 
-Name of Session:
+Name of Session: Value of Introverts
 
-Session Description (Why should students come to this session):
+Session Description (Why should students come to this session): To gain a deeper understanding of the importance of introverts
+in a extrovert valued work place.
 
-Session Outcomes (What will students learn/leave with?):
+Session Outcomes (What will students learn/leave with?): Some actionable advice to add value as an introvert.
 
 Session Outline (Please include who is facilitating and how much time you are going to spend in each section):
 
+5 min introduction / nametags
+5 min TedTalk
+20 min game
+20 min article
+20 min how to talk about adding value
+5 min famous people i.e. Elon Musk, Woz, Harrison Ford etc.
```

Comment:
Make sure to have a strong closing:  What are students taking away?  What will they do differently?